docs/directory/manifest-reference.mdx

```diff
@@ -50,18 +50,23 @@ types:
       parent: folder
       owner: user | group#member
       editor: user | group#member
-      viewer: user | group#member
+      # The "viewer" relation can be assigned to:
+      #   1. An individual "user".
+      #   2. All users (the wildcard id "*").
+      #   3. All members of a given group.
+      viewer: user | user:* | group#member
 
     # Declare permissions that are supported on "folder"
     permissions:
-      # Format: <permission_name>: relation1 | relation2 | type->permission
-      # The declaration below means that the "can_read_folder" permission
-      #   is granted by the viewer OR editor OR owner relation, OR through
-      #   a relation to the parent folder that carries the "can_read_folder"
-      #   permission.
-      can_read_folder: viewer | editor | owner | parent->can_read_folder
-      can_write_folder: editor | owner | parent->can_write_folder
       can_delete_folder: owner | parent->can_delete_folder
+      can_write_folder: can_delete_folder | parent->can_write_folder | editor
+      # Format: <permission_name>: relation_or_permission1 | relation_or_permission2 | relation->relation_or_permission2
+      # The declaration below means that the "can_read_folder" permission
+      #   is granted to:
+      #     1. anyone who can write to the folder (has the "can_write_folder" permission).
+      #     2. anyhone who can write to the parent folder (parent->can_read_folder).
```

Comment:
"anyhone" -> anyone
"who can write to the parent folder" -> "who can read from the parent folder"